### ISSUE TITLE
Published but unregistered resources are not listed in DCAT feed

### ISSUE THREAD

**Issue by peterdesmet** [oSoc15/ipt-dcat#50]
If a dataset is published, but not registered with GBIF, it will appear on the homepage (with organization `Not registered`), but it is not listed in the DCAT feed. Is this by design?

**Comment by peterdesmet** [comment]
Sorry, the use case is a bit more complicated. The dataset does not appear in the DCAT feed if **no license** has been set, which is a bit odd.

## 1. No license, not registered
1. Create dataset
2. Set organization in Basic Metadata
3. Do NOT set license in Basic Metadata
4. Do NOT register (is not possible without a license anyway)
5. Publish

=> Appears on homepage as `Not registered`, is NOT listed in DCAT feed

## 2. With license, not registered
1. Do set a license in Basic Metadata
2. Do NOT register
3. Republish

=> Appears on homepage as `Not registered`, is listed in DCAT feed (with the chosen organization as the `dct:publisher`)

## 3. With license, registered
1. Do register

=> Appears on homepage with the organization. Is listed in DCAT feed (with the chosen organization as `dct:publisher`)

**Comment by peterdesmet** [comment]
@timrobertson100 @kylebraak: is **no license** a certain threshold that makes datasets not appear in a list?

**Comment by timrobertson100** [comment]
https://github.com/gbif/ipt/blob/master/src/main/java/org/gbif/ipt/action/portal/InventoryAction.java#L35

It looks like it only lists ones that are registered (which is the intention of this utility for GBIF to inspect the holdings).

**Comment by peterdesmet** [comment]
I don't know what list of resources is being used for the DCAT feed (@simon-vc ?), but it's a list of datasets WITH a license, that are either registered or not registered with GBIF. Note that in the metadata, selecting an organization for the dataset is now mandatory, but that information is not shown on the IPT homepage until you have also actively registered the dataset (which is still optional).

**Comment by Amenai** [comment]
All of the resource published and public  : 
ResourceManager.listPublishedPublicVersions()

**Comment by Amenai** [comment]
```
/**
   * List all resources in the IPT whose last published version was public (at the time of publication). This
   * is used to populate the list of resources publicly shown on the IPT home page.
   * </br>
   * If a resource is registered with GBIF, it is assumed the resource is public and therefore is included in the list.
   * Please note only resource published using IPT v2.2 or later store a VersionHistory.
   *
   * @return list of resources, or an empty list if none were found
   */
  List<Resource> listPublishedPublicVersions(); 
```

**Comment by timrobertson100** [comment]
Does DCAT require a license?  If so, it would make sense that those datasets with none do not appear in DCAT feeds...

**Comment by peterdesmet** [comment]
No, license is recommended not required in DCAT.

**Comment by simon-vc** [comment]
A dataset doesn't have a license, only the catalog and the distributions.

**Comment by simon-vc** [comment]
I'll only use the resources that have a license, like @timrobertson100 said they use in [InventoryAction](https://github.com/gbif/ipt/blob/master/src/main/java/org/gbif/ipt/action/portal/InventoryAction.java#L35)

**Comment by timrobertson100** [comment]
Sorry - I was misunderstood... InventoryAction lists only datasets registered with GBIF.  That is more than just having a license.

**Comment by simon-vc** [comment]
I saw, there's no way to easily get it
We'll just use the public, published, licensed resources

**Comment by peterdesmet** [comment]
> We'll just use the public, published, licensed resources

Which is the implementation I tested, no? So nothing has to change, but it should be documented.

**Comment by simon-vc** [comment]
No, the previous version also published resources without license

**Comment by peterdesmet** [comment]
Indeed, you're correct.

**Comment by peterdesmet** [comment]
Confirmed fixed in `v1.2`:
- If you remove the license, the dataset is no longer listed in the DCAT feed (no need to republish even).
- If you set from public to private (and republish: required) the dataset is no longer listed in the DCAT feed.
